CONTRIBUTING.md

```diff
@@ -77,6 +77,8 @@ For all other issues and pull requests, the https://github.com/bitcoin/bitcoin m
 For global refactoring or other global changes that touche GUI code, as well as
 non-GUI code, the main repository should be used.
 
+TODO Decide what to do with interface changes (src/interfaces)
```

Comment:
If you leave the todo, point it to #19071

Reply:
I am hoping that someone comes up with a guideline here. Otherwise I will flip a coin.

Reply:
I would stick to defining interfaces in the main repo for now.